### PR TITLE
Add  Hello World 

### PR DESCRIPTION
**Pull request by ecekpo**
Set up a Hello Microverse Project in index.html file

### CODE REVIEW

File: .github/workflows [hunk 1]
```diff
@@ -1 +0,0 @@
-
```
Comment (Kossi-stack) [inline .github/workflows:1]: - [ ] kindly remove this file

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+# Hello World Project
+
+> HTML Hello World Project
+
+
+## Built With
+
+- HTML 
+- CSS
+
```
Comment (Kossi-stack) [inline README.md:11]: - [ ] Kindly remove all the empty space in the readme file.

File: README.md [hunk 3]
```diff
@@ -1 +1,37 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World Project
+
+> HTML Hello World Project
```
Comment (Kossi-stack) [inline README.md]: - [ ] kindly remove this line, this is Hello Microverse Project it is not a hello word project also add a descriptive message of the project.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
**Hi @ecekpo ,**

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages is meaningful.

## Required Changes ♻️

 - [x] Kindly edit the PR's title to something more descriptive. it can be "Hello Microverse Project PR or Hello Microverse Project".
 - [x] Kindly remove the image you put in the pull request message and put it the readme file.
 - [x] Kindly add descriptive pull request message, This helps other programmers who want to take a look at your code understand what changes you made to a particular branch.
 
 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: .github/workflows (hunk 1 above)
Comment (ecekpo) [reply .github/workflows:1]: This file is from github super linters. The instructions said we should add them.

Comment (ecekpo) [reply .github/workflows:1]: This file is from linters
Should we change the instructions?
Please check the instructions again

File: README.md (hunk 2 above)
Comment (ecekpo) [reply README.md:11]: Removing the empty space will take away the bullets

**Review by chrystalme — APPROVED** [review]
# Hi @ecekpo, 

## Project Status ✅ Approve
giphy


In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 

## Optional Requirement
- In your next project, avoid committing `.vscode/` to your repo. This makes your repo unprofessional.
- Also, be consistent with your git commit. Start every commit with a capital letter.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**